packages/aws-cdk-lib/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -394,6 +394,15 @@ export interface AwsCustomResourceProps {
    * @default - the Vpc default strategy if not specified
    */
   readonly vpcSubnets?: ec2.SubnetSelection;
+
+  /**
+   * Disables logging information related to the operation of the Lambda function. This
+   * includes displaying the event received by the handler, any API responses received,
+   * and any caught or uncaught errors
+   *
+   * @default false
+   */
+  readonly disableLogging?: boolean;
```

Comment:
A couple things in terms of this prop:
1. What if we later want to allow for levels of logging or other logging settings? This doesn't really leave room for that possibility.
2. A field that is defaulted to false and is named for the absence of something isn't typically the best way to name/default things.